lib/everypolitician/popolo/person.rb

```diff
@@ -5,6 +5,8 @@ class Error < StandardError; end
 
       attr_accessor :name, :email, :image, :gender, :birth_date, :death_date, :honorific_prefix, :honorific_suffix
 
+      attr_writer :family_name, :given_name, :identifiers, :images, :other_names, :sources
```

Comment:
I'm pretty sure we don't want `writer`s here. If this to get around the `if respond_to?("#{key}=")` in `Entity`, then I think we want to fix _that_, rather than needing to add these.